src/core_main.rs

```diff
@@ -134,7 +143,7 @@ pub fn core_main() -> Option<Vec<String>> {
         _is_quick_support |= !crate::platform::is_installed()
             && args.is_empty()
             && (arg_exe.to_lowercase().contains("-qs-")
```

Comment:
why not remove all `OR` conditions instead of hardcoding one of them as `true`?

something like:
```
_is_quick_support |= !crate::platform::is_installed()
            && args.is_empty()
            && arg_exe.to_lowercase().contains("-qs-")
```